base/deprecated.jl

```diff
@@ -1758,4 +1758,7 @@ end)
 @deprecate(SharedArray{T}(filename::AbstractString, ::Type{T}, dims::NTuple, offset; kwargs...),
            SharedArray{T,length(dims)}(filename, dims, offset; kwargs...))
 
+# LibGit2.owner -> LibGit2.repository for consistency
+eval(Base.LibGit2, :(Base.@deprecate_binding owner repository))
```

Comment:
If I'm not mistaken, I think you need `@deprecate` here rather than `@deprecate_binding`, as the latter is for types and such. So something like
```julia
eval(Base.LibGit2, :(Base.@deprecate owner(x) repository(x)))
```